test/tools/ossfuzz/Generators.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+	This file is part of solidity.
```

Comment:
Any specific reason why it's `.hpp`?

Reply:
I thought `hpp` stood for `header pre processor` or something like that and called it so since it only had preprocessor macro definitions. Just realised this is not accurate and I should probably give it to `.h` extension.